### DISCUSSION TITLE
WixToolset.Bal.wixext 5.0.2 (damaged)

### DISCUSSION THREAD

**Discussion by frnds1** [wixtoolset/issues#8852] [Questions]
After adding Bal extension to current install, regardless of the command, it always show damaged. For example:

wix extension add -g WixToolset.Bal.wixext

wix extension list
WixToolset.Bal.wixext 5.0.2 (damaged)

**Comment by robmen** [comment]
Documented change at https://wixtoolset.org/docs/fivefour/oopbas/

> WixToolset.Bal.wixext - renamed to WixToolset.BootstrapperApplications.wixext but also kept WixToolset.Bal.wixext for backwards compatibility with PackageReference in MSBuild projects. _Note: When using wix.exe, you need to use the new name WixToolset.BootstrapperApplications.wixext._

Italics mine.

**Comment by frnds1** [comment]
Thank you. Command line install works with that name.
